go.mod

```diff
@@ -2,9 +2,9 @@ module github.com/prometheus/node_exporter
 
 require (
 	github.com/beevik/ntp v0.2.0
-	github.com/coreos/go-systemd v0.0.0-20180511133405-39ca1b05acc7
+	github.com/coreos/go-systemd v0.0.0-20190204112023-081494f7ee4f
```

Comment:
It seems we could use https://github.com/coreos/go-systemd/tree/v18 instead here. The changes between v18 and the latest commit are some fixes for flaky tests.

Reply:
Yea, I think the issue is that this is not a semver tag, and it's making `go get` ignore things. There's also a broken non-semver tag on godbus.